app/views/manage/finishers/map.html.haml

```diff
@@ -87,12 +90,13 @@
         const marker = new google.maps.Marker({ ...finisher, icon, label, map, optimized: false });
 
         marker.addListener("click", () => {
-          infowindow.open({
-            anchor: marker,
-            map,
-          });
+          fetch(`http://localhost:3000${finisher.url}/card`)
```

Comment:
Can this be relative?

Reply:
Another thought, would turbo-links work here?

Reply:
Yep. My bad made the path relative.  As for turbo. I think I understand the concept of turbo but I can't figure out how to apply it to the content generated by embeded js on the map page.

Reply:
OK - let's skip that for now.